templates/garden/nightly/keras-api.libsonnet

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local base = import "base.libsonnet";
+local timeouts = import "../../timeouts.libsonnet";
+local mixins = import "../../mixins.libsonnet";
+local tpus = import "../../tpus.libsonnet";
+
+{
+  local base_command = ||| 
+    export PATH=$PATH:/root/google-cloud-sdk/bin &&
+    gcloud source repos clone cloudtpu-tf20-api-tests --project=gcp-tpupods-demo &&
+    cd cloudtpu-tf20-api-tests &&
+    pip3 install behave &&
+  |||,
+
+  local keras_test = base.GardenTest {
+    testFeature:: "aaa",
```

Comment:
just to confirm, 'aaa' is what you want here?

Reply:
Keras API first test feature name is "aaa_connection"  adding "aaa" will run that (by matching prefix), so by default I am setting that test feature, but all test cases right now override this value.  
behave framework run different test features alphabetical order, we named it with "aaa" to make sure it's first test, and that test basically check connectivity with TPU device.